nuxeo-web-ui-ftest/webdriver/test/pages/ui/group.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+export default class Group {
+
+  constructor(selector) {
+    this._selector = selector;
+  }
+
+  get el() {
+    return driver.element(this._selector);
+  }
+
+  get page() {
```

Comment:
`el` and `page` do the same. If can extend the `Base` class and just rely on `el`.